### DISCUSSION TITLE
Completly stopped working. Showing blank on chrome

### DISCUSSION THREAD

**Discussion by StrongereCommerce** [Authenticator-Extension/Authenticator#990] [Q&A]
Completely stopped working. Showing blank on chrome. See attached.
![Screenshot 2022-12-19 101125](https://user-images.githubusercontent.com/86730095/208457276-ed2a21f9-d166-441d-b1a2-3951a3eef024.png)

**Comment by memedaoo** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
